File: WebsiteConvertApp/app/src/main/java/com/eypcnn/siteyiuygulmayapma/MainActivity.java
package com.eypcnn.siteyiuygulmayapma;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.Menu;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;

public class MainActivity extends AppCompatActivity {

    private WebView eypsite;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        eypsite = (WebView) findViewById(R.id.site);
        WebSettings webSettings= eypsite.getSettings();
        webSettings.setJavaScriptEnabled(true);
        eypsite.loadUrl("http://google.com.tr");
        eypsite.setWebViewClient(new WebViewClient());
    }

    @Override
    public void onBackPressed() {
        if (eypsite.canGoBack()){
            eypsite.goBack();
        }else {
            super.onBackPressed();
        }
    }
}

File: DarkWebview/app/src/main/java/com/eypcnn/darkwebview/MainActivity.kt
package com.eypcnn.darkwebview

import android.annotation.SuppressLint
import androidx.appcompat.app.AppCompatActivity
import android.webkit.WebView
import android.os.Bundle
import android.view.View
import com.eypcnn.darkwebview.R
import com.eypcnn.darkwebview.MainActivity.WebViewClientt
import androidx.webkit.WebViewFeature
import androidx.webkit.WebSettingsCompat
import android.webkit.WebViewClient
import android.widget.Button

class MainActivity : AppCompatActivity() {

    var webView: WebView? = null
    var light: Button? = null
    var dark: Button? = null

    @SuppressLint("SetJavaScriptEnabled")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        webView = findViewById(R.id.webView)
        light = findViewById(R.id.lightweb)
        dark = findViewById(R.id.darkweb)

        webView.getSettings().javaScriptEnabled = true
        webView.loadUrl("https://www.google.com")
        webView.setWebViewClient(WebViewClientt())

        light.setOnClickListener(View.OnClickListener {
            if (WebViewFeature.isFeatureSupported(WebViewFeature.FORCE_DARK)) {
                WebSettingsCompat.setForceDark(webView.getSettings(), WebSettingsCompat.FORCE_DARK_OFF)
            }
        })

        dark.setOnClickListener(View.OnClickListener {
            if (WebViewFeature.isFeatureSupported(WebViewFeature.FORCE_DARK)) {
                WebSettingsCompat.setForceDark(webView.getSettings(), WebSettingsCompat.FORCE_DARK_ON)
            }
        })
    }

    private inner class WebViewClientt : WebViewClient() {
        override fun shouldOverrideUrlLoading(view: WebView, url: String): Boolean {
            view.loadUrl(url)
            return true
        }
    }

    override fun onBackPressed() {
        if (webView!!.canGoBack()) {
            webView!!.goBack()
        } else {
            super.onBackPressed()
        }
    }
}